config/initializers/rack_rewrite.rb

```diff
@@ -0,0 +1,7 @@
+module Openfoodnetwork
+  class Application < Rails::Application
+    config.middleware.insert_before(Rack::Lock, Rack::Rewrite) do
+      r301   '/admin/products/bulk_edit',  '/admin/products'
```

Comment:
Is this just a temporary measure to ease the transition? Or do we expect requests to bulk_edit in the future? Maybe we should add a TODO comment to remove this after a while.

Reply:
@mkllnk, I am not sure if there are bookmarks, documentation, GH issues, or any other links tied to this that would continue to request this old route. That's usually why I set these up.

I'll let your team decide on the length of time this should be here. If you decide it should be temporary I can put a TODO in here.

Reply:
Yes, these redirects are a really good practice we have been missing so far. Thank you for that. I do think a TODO comment with the current date would be nice so that a dev can remove it with more confidence in the future (probably years ;-)).

Reply:
Perfect. I'll add that comment.